src/EventExport/SortOrder.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\EventExport;
+
+use CultuurNet\UDB3\Deserializer\MissingValueException;
+
+final class SortOrder
+{
+    private string $property;
+    private string $order;
```

Comment:
In the context of this class direction is maybe a better naming?

Reply:
I made the following changes:

[Rename order in body to sort](https://github.com/cultuurnet/udb3-backend/pull/1577/commits/826be54d03cd5b8a9214cc617eca52d830c74d54)
[Rename sortOrder to sorting](https://github.com/cultuurnet/udb3-backend/pull/1577/commits/20189ab52036ccc89f27dd9ca6caac64299f60ee)
[Remove order from description in phpdoc](https://github.com/cultuurnet/udb3-backend/pull/1577/commits/6ef1caac785dcf1b7505136dde6558d4d5bd1fdf)
[Rename files test data](https://github.com/cultuurnet/udb3-backend/pull/1577/commits/aff4fcf8126c8c6d18aaf11d7fb31a2cf10842d4)